pkg/label/label.go

```diff
@@ -119,6 +124,14 @@ func New() Label {
 	}
 }
 
+// NewInitializer initialize a new Label for Jobs of TiDB initializer
+func NewInitializer() Label {
+	return Label{
+		NameLabelKey:      "initializer",
```

Comment:
Use const `InitJobLabelVal` instead. By the way, change the following backup related too.

Reply:
updated